### ISSUE TITLE
export view query problem

### ISSUE THREAD

**Issue by reinforcers** [phpmyadmin/phpmyadmin#18035]
### Describe the bug

when doing export on a view some time generate extra field statement before semicolon where it shouldn't

this is what it supposed to generate:
```sql
CREATE OR REPLACE VIEW xviewmytable  AS SELECT mytable.id AS id, mytable.personid AS personid FROM mytable WHERE (mytable.birth > '1990-01-19') GROUP BY mytable.personid  ;
```

this is what it generate:
```sql
CREATE OR REPLACE VIEW xviewmytable  AS SELECT mytable.id AS id, mytable.personid AS personid FROM mytable WHERE (mytable.birth > '1990-01-19') GROUP BY mytable.personid personid  ;
```

somehow there is double `personid` at the end of query. at the result of this, I can't directly import it, unless I  removed the last `personid` first

### Server configuration

- Operating system:
- Web server: apache/2.4.52
- Database version: mysql 8.0
- PHP version: 7.4
- phpMyAdmin version: 5.2.0

### Client configuration

- Browser: chrome 109
- Operating system: ubuntu 22.04

**Comment by williamdes** [comment]
Hi!
A lot of changes where done, it should be fixed
Can you try the latest 5.2 version in development [(phpMyAdmin 5.2+snapshot)](https://www.phpmyadmin.net/downloads/#snapshot_5.2+snapshot) also available as a [non official docker image](https://github.com/sudo-bot/docker-phpmyadmin-snapshots#readme)?

**Comment by williamdes** [comment]
Duplicate of #17321

**Comment by williamdes** [comment]
I added a test case in phpmyadmin/sql-parser@9adabf4638c35516e22f529099b410ef3f591e98

This is fixed in 5.2.1
